kolibri/core/tasks/api.py

```diff
@@ -150,7 +150,7 @@ def _enqueue_job_based_on_enqueue_args(self, registered_task, job, enqueue_args)
                 enqueue_args["enqueue_at"],
                 job,
                 queue=registered_task.queue,
-                priority=registered_task.priority,
+                priority=enqueue_args.get("priority", registered_task.priority),
```

Comment:
Excellent, this looks like the right way to handle this!

Reply:
Thank you!